cpp/src/Ice/PropertyNames.h

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) ZeroC, Inc.
 
-// Generated by makeprops.py from PropertyNames.xml, Thu Sep 26 11:13:37 2024
+// Generated by makeprops.py from PropertyNames.xml, Thu Sep 26 17:31:37 2024
```

Comment:
Unrelated to the PR: we should remove the timestamp, it adds noise to the git commits